### PR TITLE
[ci] Moved CI to Github Actions

### LINKED ISSUE #131: [CI] Move to github actions.
Move CI from travis to github actions.

### PR DESCRIPTION
**Pull request by ManishShah120**
closes #131

### CODE REVIEW

**Comment by ManishShah120** [comment]
@nemesisdesign I have tested in my fork it is working fine. In order to take effect, this has to be merged into master.

File: .github/workflows/ci.yml [hunk 1]
```diff
@@ -0,0 +1,44 @@
+---
+
```
Comment (devkapilbansal) [inline .github/workflows/ci.yml:2]: Why this extra line?

**Review by devkapilbansal — CHANGES_REQUESTED** [review]
@ManishShah120 we need to test this with single version of Python only. Therefore I don't think that we need a matrix here

**Comment by devkapilbansal** [comment]
> @nemesisdesign I have tested in my fork it is working fine. In order to take effect, this has to be merged into master.

@ManishShah120 as you are a part of `contributors` team, you should be able to create a branch on this repository itself instead of a fork. Create a new branch from master. Then you can merge and create a PR from that branch to master so that it can be tested.

File: .github/workflows/ci.yml (hunk 1 above)
Comment (ManishShah120) [reply .github/workflows/ci.yml:2]: @devkapilbansal this was unusual, I will remove it. Also will remove the matrix portion too.

**Comment by ManishShah120** [comment]
> @ManishShah120 as you are a part of `contributors` team, you should be able to create a branch on this repository itself instead of a fork. Create a new branch from master. Then you can merge and create a PR from that branch to master so that it can be tested.

Ok @devkapilbansal, then should I close this PR?

**Comment by devkapilbansal** [comment]
You can just create a new branch(say dev) from master and merge this PR to dev rather than closing it
Then open a PR from dev to master

**Pull request merged**